TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/cluster_curl/cli.py

```diff
@@ -0,0 +1,86 @@
+import json
+from typing import Dict
+import click
+import console_link.middleware.clusters as clusters_
+from console_link.environment import Environment
+
+import logging
+
+from console_link.models.cluster import HttpMethod
+
+logger = logging.getLogger(__name__)
+
+
+def split_url(url: str):
+    """Split a URL into domain and path components."""
+    parts = url.split("/", 1)
+    if len(parts) == 1:
+        return parts[0], "/"
+    else:
+        return parts[0], "/" + parts[1]
+
+
+def get_cluster_from_name(cluster_name: str, env: Environment):
+    if cluster_name == 'source_cluster':
+        return env.source_cluster
+    elif cluster_name == 'target_cluster':
+        return env.target_cluster
+    else:
+        return None
+
+
+def parse_headers(header: str) -> Dict:
+    headers = {}
+    for h in header:
+        try:
+            key, value = h.split(":", 1)
+            headers[key.strip()] = value.strip()
+        except ValueError:
+            raise click.BadParameter(f"Invalid header format: {h}. Expected format: 'Header: Value'.")
+    return headers
+
+
+@click.command()
+@click.option(
+    "--config-file", default="/etc/migration_services.yaml", help="Path to config file"
+)
+@click.option('-v', '--verbose', count=True, help="Verbosity level. Default is warn, -v is info, -vv is debug.")
+@click.option('-X', '--request', default='GET', help="HTTP method to use",
```

Comment:
--request is a bit confusing.  Is that what curl (or anything else) uses?  'Verb' or 'method' are common labels

Reply:
This is what curl uses